src/haddock/modules/topology/topoaa/defaults.cfg

```diff
@@ -10,8 +10,7 @@ limit = true
 # tolerance in percentage
 tolerance = 0
 
-[input]
-[input.mol1]
+[mol1]
```

Comment:
So no need for `topoaa` in the param name here? Will be confusing to users since our example config files do use the `topoaa.molX` naming.  To me the default.cfg file of a module shows me all the possible parameters and the way to define those. I think we already discussed that before since we have the same inconsistency before these changes.

Reply:
It's a different thing. In the `defaults.cfg` we use `[mol1]` and not `[topoaa.mol1]` because the `default.cfg` itself is the `topoaa` module, while the user config has several modules so we need to define the `[topoaa]` layer. Keep in mind that the user will **never** access the `defaults.cfg` *per se*. We will always provide a script or an API that translates the `defaults.cfg` to what the user needs. I see in #274 I've create an `haddock3-cfg` CLI for that purpose. Actually, after #274 is merged the concern you raised in your comment get's solved because we move from the defaults from `cfg` to `yml`.